{
  "uid" : "cd7e2443819d9c8",
  "name" : "test_post_body_negative[backup_POST_negative_Invalid field types and values]",
  "fullName" : "api.test_request_body.test_post#test_post_body_negative",
  "historyId" : "36ae51f5ac1ce92b89463d137a1087d1",
  "time" : {
    "start" : 1618389424738,
    "stop" : 1618389424738,
    "duration" : 0
  },
  "description" : "\n    Negative cases of request body testing\n    Includes sets of invalid field values - out of boundary values,\n    nullable and required if not possible, fields with incorrect types etc.\n    ",
  "descriptionHtml" : "<pre><code>Negative cases of request body testing\nIncludes sets of invalid field values - out of boundary values,\nnullable and required if not possible, fields with incorrect types etc.\n</code></pre>\n",
  "status" : "failed",
  "statusMessage" : "AssertionError: Expecting status code 201 but got 422",
  "statusTrace" : "request = <SubRequest 'prepare_post_body_data' for <Function test_post_body_negative[backup_POST_negative_Invalid field types and values]>>\napp_fs = <tests.utils.api_objects.APPApi object at 0x7fd2f4238b80>\n\n    @pytest.fixture()\n    def prepare_post_body_data(request, app_fs: APPApi):\n        \"\"\"\n        Fixture for preparing test data for POST request, depending on generated test datasets\n        \"\"\"\n        test_data_list: List[TestDataWithPreparedBody] = request.param\n>       valid_request_data = DbFiller(app=app_fs).generate_valid_request_data(\n            endpoint=test_data_list[0].test_data.request.endpoint, method=Methods.POST\n        )\n\ntests/api/test_request_body/test_post.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_data/db_filler.py:39: in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\ntests/test_data/db_filler.py:80: in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\ntests/test_data/db_filler.py:108: in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\ntests/test_data/db_filler.py:88: in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\ntests/utils/api_objects.py:72: in exec_request\n    status_code_should_be(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [422]>, status_code = <HTTPStatus.CREATED: 201>\n\n    @allure.step(\"Response status code should be equal {status_code}\")\n    def status_code_should_be(response: Response, status_code=HTTPStatus.OK):\n        \"\"\"Assert response status code\"\"\"\n>       assert (\n            response.status_code == status_code\n        ), f\"Expecting status code {status_code} but got {response.status_code}\"\nE       AssertionError: Expecting status code 201 but got 422\n\ntests/steps/asserts.py:16: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "app_fs",
    "time" : {
      "start" : 1618389424745,
      "stop" : 1618389426222,
      "duration" : 1477
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Run container: local/app:BapDGBzpjC",
      "time" : {
        "start" : 1618389424763,
        "stop" : 1618389425177,
        "duration" : 414
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "prepare_post_body_data",
    "time" : {
      "start" : 1618389426222,
      "stop" : 1618389426329,
      "duration" : 107
    },
    "status" : "failed",
    "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/api/test_request_body/test_post.py\", line 32, in prepare_post_body_data\n    valid_request_data = DbFiller(app=app_fs).generate_valid_request_data(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 39, in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 80, in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 108, in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 88, in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
    "steps" : [ {
      "name" : "Generate valid request data",
      "time" : {
        "start" : 1618389426222,
        "stop" : 1618389426329,
        "duration" : 107
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 39, in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 80, in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 108, in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 88, in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
      "steps" : [ {
        "name" : "Send LIST /cluster/",
        "time" : {
          "start" : 1618389426222,
          "stop" : 1618389426276,
          "duration" : 54
        },
        "status" : "passed",
        "steps" : [ {
          "name" : "Response status code should be equal <HTTPStatus.OK: 200>",
          "time" : {
            "start" : 1618389426276,
            "stop" : 1618389426276,
            "duration" : 0
          },
          "status" : "passed",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "response",
            "value" : "<Response [200]>"
          }, {
            "name" : "status_code",
            "value" : "<HTTPStatus.OK: 200>"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : false,
          "hasContent" : true
        } ],
        "attachments" : [ {
          "uid" : "808ae90c3d10a63f",
          "name" : "Full request log",
          "source" : "808ae90c3d10a63f.txt",
          "type" : "text/plain",
          "size" : 504
        } ],
        "parameters" : [ ],
        "stepsCount" : 1,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "Send POST /cluster/",
        "time" : {
          "start" : 1618389426276,
          "stop" : 1618389426328,
          "duration" : 52
        },
        "status" : "failed",
        "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
        "statusTrace" : "  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
        "steps" : [ {
          "name" : "Response status code should be equal <HTTPStatus.CREATED: 201>",
          "time" : {
            "start" : 1618389426328,
            "stop" : 1618389426328,
            "duration" : 0
          },
          "status" : "failed",
          "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "response",
            "value" : "<Response [422]>"
          }, {
            "name" : "status_code",
            "value" : "<HTTPStatus.CREATED: 201>"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ {
          "uid" : "7bd9118556fd7eb1",
          "name" : "Full request log",
          "source" : "7bd9118556fd7eb1.txt",
          "type" : "text/plain",
          "size" : 769
        } ],
        "parameters" : [ ],
        "stepsCount" : 1,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "endpoint",
        "value" : "<Endpoints.Backup: Endpoint(path='backup', methods=[<Methods.GET: Method(function=<function get at 0x7fd2f4605a60>, url_template='/{name}/{id}/', default_success_code=<HTTPStatus.OK: 200>)>, <Methods.LIST: Method(function=<function get at 0x7fd2f4605a60>, url_template='/{name}/', default_success_code=<HTTPStatus.OK: 200>)>, <Methods.POST: Method(function=<function post at 0x7fd2f4605c10>, url_template='/{name}/', default_success_code=<HTTPStatus.CREATED: 201>)>], data_class=<class 'tests.utils.data_classes.BackupFields'>)>"
      }, {
        "name" : "method",
        "value" : "<Methods.POST: Method(function=<function post at 0x7fd2f4605c10>, url_template='/{name}/', default_success_code=<HTTPStatus.CREATED: 201>)>"
      } ],
      "stepsCount" : 4,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "cmd_opts",
    "time" : {
      "start" : 1618389387072,
      "stop" : 1618389387072,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "image",
    "time" : {
      "start" : 1618389387072,
      "stop" : 1618389391106,
      "duration" : 4034
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Run container: app:refs_pull_6_merge",
      "time" : {
        "start" : 1618389387117,
        "stop" : 1618389387769,
        "duration" : 652
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "app_fs::fin",
    "time" : {
      "start" : 1618389426350,
      "stop" : 1618389426647,
      "duration" : 297
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Gather /var/www/html/storage/app/ from APP container: 90364bce7dec0fbf7ee60e60016fef321d4208438faece9bf9c4c05a4fb60f64",
      "time" : {
        "start" : 1618389426350,
        "stop" : 1618389426417,
        "duration" : 67
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "301c17db27c1499a",
        "name" : "test_post_body_negative[backup_POST_negative_Invalid field types and values]_1618389426.3499668.tgz",
        "source" : "301c17db27c1499a.tgz",
        "type" : "application/gzip",
        "size" : 1192
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "image::fin",
    "time" : {
      "start" : 1618389426648,
      "stop" : 1618389426659,
      "duration" : 11
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "API"
  }, {
    "name" : "subSuite",
    "value" : "POST"
  }, {
    "name" : "suite",
    "value" : "Body tests"
  }, {
    "name" : "tag",
    "value" : "negative"
  }, {
    "name" : "host",
    "value" : "fv-az252-672"
  }, {
    "name" : "thread",
    "value" : "4926-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "api.test_request_body.test_post"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "prepare_post_body_data",
    "value" : "[TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43221c0, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43222e0, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43223d0, test_body={'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43224c0, test_body={'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322580, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43226a0, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322790, test_body={'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322880, test_body={'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)})]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "negative" ]
  },
  "source" : "cd7e2443819d9c8.json",
  "parameterValues" : [ "[TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43221c0, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43222e0, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43223d0, test_body={'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43224c0, test_body={'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322580, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f43226a0, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322790, test_body={'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST backup and expect 422 status code. at 0x7fd2f4322880, test_body={'name': PreparedFieldValue(value='nooChGIPYJSyxagwnOCjpSkGDSFrQsMvcNOuBZVDYDCyBUfMQfaeZGEKYgGxrCKDpvmdZrxPpibkcLHPNKoMsbPdSHGvQzpplgqZJFxrZrTKNxFOchLDTubXshiSkyMCisynOCnNVGOZsMncaFesXDWpJaOGbPEKuAJMppFNFlxgqwQzIQXHLYipYynKbeFoBxfjRrvpofgSIDZAlPSrtHViCcjqlaxUkFlFpfEQziANPholccIKuvjmKkTZOQam', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'HwGQTzzTuhUheOpdvVzychcRsBuYSSkYyvMbLGwHAeQNBdPPZsuRHSTfCCusalahWtgUinOCZpHkfKzkKtpxuDIgQXEAhqSXEJiCQVVKhYuFicFzKDdQHRYSTgdgGVYpmfYTZrrNjQathtpyxFlJKBjFwOMAyzYzoLQlxqyxPWXEGsMtqVpsAnFFctmoKjTeZgMHKnBzewiInEPZBeRlySCBFmtFQkiEitmURKNvaVEJdZDmjispqPhspuoKhEm'], _sp_vals_negative=[...]), drop_key=False)})]" ]
}